go/obscuronode/enclave/core/crypto.go

```diff
@@ -30,12 +30,6 @@ func DecryptTx(tx nodecommon.EncryptedTx) nodecommon.L2Tx {
 	return t
 }
 
-// EncryptResponse - encrypts the response from the evm with the viewing key of the sender
-func EncryptResponse(resp []byte) nodecommon.EncryptedResponse {
-	// TODO
-	return resp
-}
-
 // DecryptResponse - the reverse of EncryptResponse
 func DecryptResponse(r nodecommon.EncryptedResponse) []byte {
```

Comment:
if the EncryptResponse function was removed, shouldn't the decrypt go as well. i assume it was replaced with something else

Reply:
I assumed this is where the encryption of messages _to_ the enclave would go? Remember that we haven't implemented that yet.

Reply:
this was supposed to be used in the tests

Reply:
Ok, thanks. Will remove.